src/test/ui/parser/attr.stderr

```diff
@@ -14,12 +13,5 @@ LL - #![lang = "foo"]
 LL + #[lang = "foo"]
    | 
 
-error[E0522]: definition of an unknown language item: `foo`
```

Comment:
This error doesn't appear anymore since I filter out any (unexpected) inner attributes. I hope this is okay. I can try to find another way of removing the "expected item after attribute" error.